src/components/CarInfo.component.tsx

```diff
@@ -0,0 +1,168 @@
+import React, { useContext, useEffect, useState } from "react";
+import styled from "styled-components";
+import { defaultTheme } from "../theme";
+import { FlexColumn, FlexRow, H3, H4, P } from "./styled";
+import { CarRowInfo } from "../interfaces/Car";
+import { UserContext } from "../contexts/UserContext";
+import { useNavigate } from "react-router-dom";
+import { PATHNAME } from "../utils";
+import { CREATE_USER_CAR, DELETE_USER_CAR } from "../graphql/mutations";
+import { useMutation } from "@apollo/client";
+import { deleteCarVariables } from "../graphql/variables";
+import { FavoriteButton } from "./FavoriteButton.component";
+import { ToastContainer, toast } from "react-toastify";
+import "react-toastify/dist/ReactToastify.css";
+export const CarInfo = ({
```

Comment:
please use blank line after all the imports statements, again you can check how to configure this with your linter rules

Reply:
fixed! thanks!